keras_nlp/models/deberta_v3/deberta_v3_masked_lm.py

```diff
@@ -145,6 +146,13 @@ def __init__(
         self.backbone = backbone
         self.preprocessor = preprocessor
 
+        self.compile(
+            loss=keras.losses.SparseCategoricalCrossentropy(from_logits=True),
+            optimizer=keras.optimizers.Adam(5e-5),
+            metrics=keras.metrics.SparseCategoricalAccuracy(),
```

Comment:
I believe this should be `weighted_metrics` as we pass sample weights in the preprocessing.